code_challenges.rb

```diff
@@ -22,14 +32,59 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# I want to look through the characters in the single string and check for vowels
+# I think there is a specific method in ruby that would allow us to do that
+# After doing some research on google I found the method delete which we could pass in vowels as arguments
+
+def delete_vowels string
+    string.delete('aeiouAEIOU');
+end
 
+# p delete_vowels album1
+# p delete_vowels album2
+# p delete_vowels album3
 
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a get_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
+class Bike 
+    attr_reader :model, :wheels, :current_speed
```

Comment:
good use of attr_reader if you don't want your user to change these values directly.